nemo/collections/nlp/modules/common/megatron/utils.py

```diff
@@ -417,8 +417,8 @@ def split_list(inputs, num_chunks):
     """
     Split a list into equal sized chunks
     """
+    # if len(inputs) % chunk_size != 0, round down the chunk size
```

Comment:
remove this line if it's not used

Reply:
not used but try to provide extra explanation on what happens